_sass/base/_markdown.scss

```diff
@@ -0,0 +1,92 @@
+.markdown {
+  padding-bottom: 60px;
+  
+  // Text Blocks
+  p {
+    margin: 10px 0;
+    font-weight: 300;
+
+    @media (min-width: $tablet) {
+      margin: 20px 0;
+    }
+  }
+  
+
+  img {
+    width: 100%;
+  }
+
+  // List Items
+  li {
+    line-height: 24px;
+  }
+
+  // Lists
+  ol,
+  ul {
+    list-style: initial;
+    list-style-position: inside;
+    color: $black;
+    font-size: 15px;
+    margin: 0 20px;
+
+    @media (max-width: 768px) {
+      margin: 0;
+    }
+  }
+
+  h1 {
+    color: $secondBrandColor;
+  }
+
+  // Section Headlines 
+  h2 {
+    margin-bottom: 20px; 
+    margin-top: 30px;
+  }
+
+  // Section Headlines 
+  h3 {
+    color: $black;
+    font-size: 18px;
+    font-weight: 500;
+    letter-spacing: 1px;
+    margin-bottom: 20px; 
+    margin-top: 30px;
+  }
+
+  // Section Subheadlines
+  h4 {
+    margin: 20px 25px;
+    font-size: 16px;
+    font-weight: 300;
+    color: $dark-gray;
+    
+    @media (max-width: 768px) {
+      margin: 20px 0;
+    } 
+    // Text immediately following h6 should have same level of indention  
```

Comment:
This looks a bit strange in preview, largely because the indentation is at the same level as code blocks so if there's a code block after the first paragraph (as there is for almost all the h4s I've checked) it looks like a huge chunk of text has inexplicably moved sideways - eg http://wildebeestdev.github.io/grpc.github.io/docs/tutorials/basic/java.html#calling-service-methods